streams-activitypub-graph/src/test/scala/org/apache/streams/activitypub/graph/test/cases/WebfingerGraphImplTest.scala

```diff
@@ -0,0 +1,126 @@
+package org.apache.streams.activitypub.graph.test.cases
+
+import org.apache.http.HttpStatus
+import org.apache.http.client.utils.URIBuilder
+import org.apache.http.entity.ContentType
+import org.apache.streams.activitypub.api.pojo.WebfingerQueryRequest
+import org.apache.streams.activitypub.api.pojo.WebfingerQueryResponse
+import org.apache.streams.activitypub.graph.impl.WebfingerGraphImpl
+import org.apache.streams.activitypub.graph.test.ActivityPubGraphTestSuite
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.slf4j.LoggerFactory
+
+import java.nio.charset.Charset
+import scala.util.Try;
+
+class WebfingerGraphImplTest {
+
+    private final val LOGGER = LoggerFactory.getLogger(classOf[GraphDatabaseServerAvailableTest]);
+
+    val primaryTestResourceURI = "https://mastodon.social/users/steveblackmon"
```

Comment:
the project doap file has my mastodon link in it, will likely parameterize this and similar test cases with URIs sourced from a test scope config file as the test dataset(s) grow and evolve